src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -307,7 +307,15 @@ type SentryEventBase = {
     enhancements: string;
   };
 
+  userReport?: any;
+
   crashFile: EventAttachment | null;
+
+  sdk?: {
+    name?: string;
+  };
+
+  sdkUpdates?: any;
```

Comment:
Do all event types have these properties?

Reply:
transaction events and error events have them both, yea